src/client/voice/receiver/PacketHandler.js

```diff
@@ -78,6 +79,23 @@ class PacketHandler extends EventEmitter {
     const ssrc = buffer.readUInt32BE(8);
     const user = this.userFromSSRC(ssrc);
     if (!user) return;
+
+    if (this.speakingTimeouts.get(ssrc)) {
+      clearTimeout(this.speakingTimeouts.get(ssrc));
+      this.speakingTimeouts.delete(ssrc);
+    } else {
+      this.connection.onSpeaking({ user_id: user.id, ssrc: ssrc, speaking: 1 });
+    }
+    let speakingTimer = setTimeout(() => {
```

Comment:
```suggestion
    const speakingTimer = setTimeout(() => {
```

speakingTimer is never reassigned, so `const` should be used for styling consistency